src/pages/home/report/ReportActionItemCreated.js

```diff
@@ -47,7 +50,36 @@ const defaultProps = {
     policy: {},
 };
 
+const IMAGE_OFFSET_Y = 75;
+const ANIMATION_BOOST = 1.3;
+
 function ReportActionItemCreated(props) {
+    const {windowWidth, isSmallScreenWidth} = useWindowDimensions();
+    const IMAGE_OFFSET_X = windowWidth / 2;
+
+    // Get data from phone rotation sensor and prep other variables for animation
+    const animatedSensor = useAnimatedSensor(SensorType.GYROSCOPE);
+    const xOffset = useSharedValue(0);
+    const yOffset = useSharedValue(0);
+
+    // Apply data to create style object
+    const animatedStyles = useAnimatedStyle(() => {
+        /*
+         * We use x and y gyroscope velocity and add it to position offset to move background based on device movements.
+         * Position the phone was in while entering the screen is the initial position for background image.
+         */
+        const {x, y} = animatedSensor.sensor.value;
+        // The x vs y here seems wrong but is the way to make it feel right to the user
+        xOffset.value = NumberUtils.clampWorklet(xOffset.value - y * ANIMATION_BOOST, -IMAGE_OFFSET_X, IMAGE_OFFSET_X);
```

Comment:
I think we should switch this:

```suggestion
        xOffset.value = NumberUtils.clampWorklet(xOffset.value + y, -IMAGE_OFFSET_X, IMAGE_OFFSET_X);
```

Such that if you're holding the phone flat and tilt it to the left, the animation shifts to the right. This creates a better perspective effect imo and is more inline with how it behaves in the Y direction